std_msgs/Float64MultiArray bin_centroid
std_msgs/Float64MultiArray bin_corners
std_msgs/Float64 bin_num
std_msgs/Int16 obj_id
geometry_msgs/Point obj_centroid
geometry_msgs/Point obj_normal
geometry_msgs/Point obj_axis
geometry_msgs/Point gripping_centroid
geometry_msgs/Point gripping_normal
geometry_msgs/Point gripping_axis
std_msgs/Bool flag_grip
std_msgs/Int16 obj_psn_suction
---
std_msgs/Float64MultiArray jts_reach_tote
std_msgs/Float64MultiArray jts_back_tote
std_msgs/Float64MultiArray jts_reach_rack
std_msgs/Bool flag_grip


